src/Components/DropdownField.res

```diff
@@ -24,13 +34,37 @@ let make = (
     let target = ev->ReactEvent.Form.target
     let value = target["value"]
     setValue(value)
+    if isDisplayValueVisible {
+      let findDisplayValue =
+        options
+        ->Array.find((ele: optionType) => ele.value === value)
+        ->Option.getOr({
+          label: "",
```

Comment:
Please create a default object of this and use it everywhere in the getOr

Reply:
Updated